recipe/meta.yaml

```diff
@@ -80,7 +77,7 @@ outputs:
     script: install_python.bat  # [win]
     requirements:
       run:
-        - {{ pin_subpackage("gmsh", max_pin="x.x.x") }}
+        - {{ pin_subpackage("gmsh", exact=True) }}
```

Comment:
this change helps in multi-output recipes avoid co-installation of packages with different build numbers.